plugins/other/pacman

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+# DESCRIPTION
+#
+# This plugin monitors the number of pending updates in pacman.
+#
+# WARNINGS
+#
+# This plugin will issue a warning state whenever any package can be updated.
+#
+# ENVIRONMENT
+#
+# env.WARNING The number of updates at which a warning will be issued. If not
```

Comment:
I would suggest to use a more descriptive environment variable name here. Maybe some like `outdated_packages_warning_threshold`. Lower case names are common for environment variables used in munin plugins.